CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.0-pnc0] - 2025-08-13
+
+### Added
+
+- A PNC-customized branch of the FastMatch pipeline with specific metadata fields and parameters tailored to PNC requirements.
```

Comment:
Could you add the PR number here?

Reply:
Updated in 5436b29938939a52c7db39c92f488ae9df8e5272